buildpackage/config_reader.go

```diff
@@ -117,6 +121,17 @@ func (r *ConfigReader) Read(path string) (Config, error) {
 	return packageConfig, nil
 }
 
+func (r *ConfigReader) ReadBuildpackDescriptor(path string) (dist.BuildpackDescriptor, error) {
```

Comment:
Note, pkg/buildpack/buildpack.go has `FromBuildpackRootBlob` which also reads the descriptor, IDK if any of that code can (or should) be shared